Tests/InternalTests/PresenceDataTests.swift

```diff
@@ -26,11 +26,13 @@ class PresenceDataTests: XCTestCase {
         jsonString = "{\"type\":\"PUBLISHER\",\"rawLocations\":true}"
         data = try PresenceData.fromJSONString(jsonString)
         XCTAssertEqual(data.type, .publisher)
+        // swiftlint:disable:next xct_specific_matcher
```

Comment:
Any specific reason why we're ignoring this rule here, instead of using `XCTAssertTrue`, and `XCTAssertFalse` below?

Reply:
Yep — it's because `data.rawLocation` is a `Bool?` and hence can't be used with `XCTAssertTrue/False`, but SwiftLint doesn’t realise this.

Reply:
Good idea, changed it :)